samtranslator/model/eventsources/push.py

```diff
@@ -788,6 +788,35 @@ def _add_swagger_integration(self, api, function, intrinsics_resolver):
                     path=self.Path, method_name=self.Method, request_model=self.RequestModel
                 )
 
+                validate_body = self.RequestModel.get("ValidateBody")
+                validate_request = self.RequestModel.get("ValidateRequest")
+
+                # Checking if any of the fields are defined as it can be false we are checking if the field are not None
+                if not isinstance(validate_body, type(None)) or not isinstance(validate_request, type(None)):
+
+                    # as we are setting two different fields we are here setting as default False
+                    # In case one of them are not defined
+                    validate_body = False if type(validate_body) is type(None) else validate_body
```

Comment:
Same here and below, you can use `validate_body is None`.

Reply:
Done, sometimes being fancy is not good idea, tbh I always use `is None`. I think due the size of this project I tried to be too much fancy 😅. Jokes aside, fixed!